### PR TITLE
i18n(zh-cn): Translate images.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate images.mdx
- https://github.com/withastro/docs/pull/3739

### CODE REVIEW

**Comment by sarah11918** [comment]
So excited for this @liruifengv !

Heads up that I made another commit with some changes to three of the `migrate-to/from-` guides: `https://github.com/withastro/docs/pull/3739/commits/58feb02f562be4bd0f113617b287ec25614079d7`

Then, there were some small conflict fixes in the following commits.

**Comment by liruifengv** [comment]
> So excited for this @liruifengv !↳
> 
> Heads up that I made another commit with some changes to three of the `migrate-to/from-` guides: `https://github.com/withastro/docs/pull/3739/commits/58feb02f562be4bd0f113617b287ec25614079d7`↳
> 
> Then, there were some small conflict fixes in the following commits.↳

Done!

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:321]:
```suggestion
根据情况在你的 React 组件中将 Gatsby 的 `<StaticImage />` 和 `<GatsbyImage />` 组件转换为 [Astro 自己的图像集成组件](/zh-cn/guides/images/#image--astroassets) 或者转化成 [标准的 HTML `<img>` / JSX `<img />` 标签](/zh-cn/guides/images/#框架组件中的图像)。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:332]:
```suggestion
Astro 的 `<Image />` 组件只能在 `.astro` 和 `.mdx` 文件中工作。查看该组件可用的[完整组件属性列表](/zh-cn/guides/images/#属性)，并注意它与 Gatsby 的属性有一些不同。
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (100gle) [inline src/content/docs/zh-cn/guides/images.mdx:57]:
```suggestion
:::tip[什么是累积布局偏移（Cumulative Layout Shift）？]
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx
Comment (100gle) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:334]:
```suggestion
要继续使用标准 Markdown 语法 (`![]()`) [在 Markdown (`.md`) 文件中使用图像](/zh-cn/guides/images/#markdown-文件中的图像)，你可能需要更新链接。对于本地图像，不支持在 `.md` 文件中直接使用 HTML `<img>` 标签，必须转换为 Markdown 语法。
```

**Review by 100gle — APPROVED** [review]
有几个小地方排版格式稍微修改一下，其他没有什么大问题

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:141]:
```suggestion
当以原始宽高比使用本地图像时，`width` 和 `height` 可以从源文件中自动推断，并且它们是可选的。
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:154]:
```suggestion
- 一个预设值（`low`、`mid`、`high`、`max`），可以在不同格式之间自动标准化。
```

**Review by 100gle — APPROVED** [review]
LGTM

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:259]:
```suggestion
当你不能使用 `<Image />` 组件时，请使用 HTML `<img>` 元素，例如：
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:266]:
```suggestion
你可以使用 [`image.domains`](/zh-cn/reference/configuration-reference/#imagedomains) 和 [`image.remotePatterns`](/zh-cn/reference/configuration-reference/#imageremotepatterns) 配置授权图像源 URL 域和模式列表，以进行图像优化。这个配置是一个额外的安全层，用于在显示来自外部源的图像时保护你的站点。
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:322]:
```suggestion
如果你需要对图像属性做更多的控制，我们建议使用 `.mdx` 文件格式，除了 Markdown 语法，你还可以使用 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:626]:
```suggestion
如果你需要对图像属性做更多的控制，我们建议使用 `.mdx` 文件格式，除了 Markdown 语法，你还可以使用 Astro 的 `<Image />` 组件或 JSX `<img />` 标签。使用 [MDX 集成](/zh-cn/guides/integrations-guide/mdx/) 为 Astro 添加对 MDX 的支持。
```

File: src/content/docs/zh-cn/guides/images.mdx
Comment (kanweiwei) [inline src/content/docs/zh-cn/guides/images.mdx:612]:
```suggestion
现在可以使用标准的 Markdown `![alt](src)` 语法，在 Markdown、MDX 和 Markdoc 中引用项目 `src/` 中的相对图像。
```

**Review by kanweiwei — APPROVED** [review]
LGTM

**Review by sarah11918 — APPROVED** [review]
I see a bunch of LGTM's, so I'm approving, but waiting for the green light from @Yan-Thomas before merging!

Excellent work @liruifengv and thank you @100gle and @kanweiwei !  :raised_hands:

**Pull request merged**
